### PR TITLE
Add missing condition for Organizations

### PR DESCRIPTION
**Pull request by spencerhcheng**
### Proposed changes

The deactivation requirements are incomplete for the Organization section. An organization may not have additional members or owners, else the deactivation will not be permitted.

### Related issues (optional)

https://github.com/docker/mercury-ui/pull/2224

### CODE REVIEW

File: docker-hub/deactivate-account.md [hunk 1]
```diff
@@ -48,5 +48,7 @@ Before deactivating an organization, please complete the following:
 
 Once you have completed all the steps above, you may deactivate your organization. On Docker Hub, go to **Organizations** > **_Your Organization_** > **Settings** > **Deactivate Org**.
 
+4. Remove all other members, including those in the **Owners** team, within the organization.
```
Comment (thaJeztah) [inline docker-hub/deactivate-account.md:51]: Should this step be above the note above, or below the one below? It feels like those belong together (but not familiar with the exact steps)

Comment (spencerhcheng) [reply docker-hub/deactivate-account.md]: @thaJeztah good point, thanks for the catch. Yes, it should be above. I moved it to step 3 and

```
Once you have completed all the steps above, you may deactivate your organization. On Docker Hub, go to **Organizations** > **_Your Organization_** > **Settings** > **Deactivate Org**.

> This cannot be undone! Be sure you've gathered all the data you need from your organization before deactivating it.
{: .warning 
```
remains below the check list.

**Comment by thaJeztah** [comment]
Thanks! Changes SGTM, but perhaps someone else from hub can verify

**Review by siarheikarneichyk — APPROVED** [review]

**Comment by spencerhcheng** [comment]
@thaJeztah are you able to merge this? I'm not authorized to.

**Review by thaJeztah — APPROVED** [review]
LGTM, thanks!

**Pull request merged**